src/components/TitleTransferPanel/AddressCta/AddressCta.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { SvgIcon, SvgIconBook, SvgIconX } from "../../Helpers/SvgIcon";
+import { ButtonIcon, ButtonSolid, ButtonCircle } from "../../Helpers/Button";
+
+interface AddressCtaProps {
+  holderAddress: string;
+  beneficiaryAddress?: string;
+  isDisabledAddressBook?: boolean;
+  buttonText: string;
+  onButtonSubmit?(event: React.MouseEvent<HTMLButtonElement>): void;
+}
+
+export const AddressCta = ({
+  holderAddress,
+  beneficiaryAddress,
+  isDisabledAddressBook,
+  buttonText,
+  onButtonSubmit
+}: AddressCtaProps) => {
+  const isDisableButtonSubmit =
+    beneficiaryAddress !== undefined
+      ? holderAddress.length === 0 || beneficiaryAddress.length === 0
+      : holderAddress.length === 0;
```

Comment:
This is so hard to read or understand, can explain what are you trying to do here?

Reply:
If I'm guessing it right, you want all fields to be filled right? How about another 

```
const isSubmitEnabled = beneficiaryAddress != undefined ? beneficiaryAddress && holderAddress : holderAddress;
```

Reply:
eh i need the result to be boolean otherwise the disabled props in ts tak match, i edited abit